orion/README.md

```diff
@@ -194,26 +195,29 @@ Great job getting here! You've built 2 local webservers to host the web applicat
 
 To make it easier on yourself in the future and to develop faster, you can use this script here in the `orion` folder.
 Before doing so, close the other Terminal tabs so we don't end up creating multiple web servers with conflicting port numbers.
+
 *Mac users only*
 ```
 ./run_admin_local.sh
 ```
 *Windows Users only*
+
+Open Windows PowerShell and run the command above. Alternatively, on Command Prompt you can use
 ```
-coming soon...
+run_admin_local.sh
```

Comment:
hmm I'm not sure about this. Windows PowerShell is only default for Windows 10. And normally, you can't run .sh files on Windows. I'd say keep the note about Windows PowerShell above. But, if viewers don't have Windows 10, maybe I'll create a .bat or .cmd file for them to run on Command Prompt.

Reply:
I think it works for me because it opens a git bash application. I added a batch file that works for Command Prompt.

Reply:
Now that the batch file is in, could you please change this to `run_admin_local.bat`?

Reply:
ah nvm you did.